File: Unidad 3/Java/Programa8U3.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 */

package com.mycompany.programa8u3;

import java.util.Scanner;
public class Programa8U3 {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int num1, num2;
        System.out.println("Programa para saber cual numero es mayor");
        System.out.println("Introdusca el primer numero");
        num1 = sc.nextInt();
        System.out.println("Introdusca el segundo numero");
        num2 = sc.nextInt();
        
        if(num1 > num2){
             System.out.println("El numero "+num1+" es mayor que "+num2);
        }else{
             System.out.println("El numero "+num2+" es menor que "+num1);
        }
        
        
    }
}

File: Unidad 3/C#/Programa8U3.cs
// See https://aka.ms/new-console-template for more information

using System;

namespace MyApp // Note: actual namespace depends on the project name.
{
    internal class Programa8U3
    {
        static void Main(string[] args)
        {
            int num1, num2;
            Console.WriteLine("Programa para saber cual numero es mayor");
            Console.WriteLine("Introdusca el primer numero");
            num1 = Convert.ToInt32(Console.ReadLine());
            Console.WriteLine("Introdusca el segundo numero");
            num2 = Convert.ToInt32(Console.ReadLine());

            if (num1 > num2)
            {
                Console.WriteLine("El numero " + num1 + " es mayor que " + num2);
            }
            else
            {
                Console.WriteLine("El numero " + num2 + " es menor que " + num1);
            }
        }
    }
}